duckduckgo/Android
# Remove tracker blocker opt in experiment

Task/Issue URL: https://app.asana.com/0/4603919660411/1134061660765792
Tech Design URL: 
CC: 

**Description**:


**Steps to test this PR**:
From a fresh install, verify that there are no problems navigating onboarding.
1. From a device which supports prompting for default app (>= Nougat), check the onboarding flow, and that there is no tracker-blocker opt-in prompts
1. From a device which does not support prompting for default app (< Nougat), check the onboarding flow, and that there is no tracker-blocker opt-in prompts



---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
